### PR TITLE
adding litesata

### PR DESCRIPTION
**Pull request by timkpaine**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by timkpaine** [comment]
@conda-forge/help-python all set!

File: recipes/litesata/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    - liteiclink >=2023.08
+    - migen
+    - packaging
+    - pyyaml
+
+test:
+  imports:
+    - litesata
+  commands:
+    - litesata_gen --help
```
Comment (BastianZim) [inline recipes/litesata/meta.yaml:37]:
```suggestion
    - litesata_gen --help
    - pip check
requires:
    - pip
```

File: recipes/litesata/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{% set name = "litesata" %}
+{% set version = "2023.12" %}
+{% set sha256 = "469570e970a8b4e585d50597afae77a52ace638990eefaea89f9d2ab1bb2ee18" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
```
Comment (BastianZim) [inline recipes/litesata/meta.yaml:10]:
```suggestion
```

**Review by BastianZim — CHANGES_REQUESTED** [review]
Just one small thing.

**Comment by timkpaine** [comment]
Thanks @bastianzim but id like to keep this in line with prior packages that are the same:
- https://github.com/conda-forge/staged-recipes/pull/26117
- https://github.com/conda-forge/staged-recipes/pull/26093
- https://github.com/conda-forge/staged-recipes/pull/22532
- https://github.com/conda-forge/staged-recipes/pull/22508
- https://github.com/conda-forge/staged-recipes/pull/22507
- https://github.com/conda-forge/staged-recipes/pull/22332

Also for pip check see:
- https://github.com/conda-forge/staged-recipes/pull/26117#discussion_r1581899783
- https://github.com/conda-forge/staged-recipes/pull/26093#discussion_r1597173893

**Pull request merged**